examples/golang/components/http-client-tinygo/wit/world.wit

```diff
@@ -1,6 +1,9 @@
 package wasmcloud:httpclient;
 
 world hello {
+  // tinygo
```

Comment:
nit: did TinyGo need this? or was this just a leftover comment

Reply:
unfortunately tinygo requires that `import` to be present, and was looking for a way to express "hey, this is tinygo specific"